docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+.. To find out more about how Flocker manages migration of containers with volumes, see *add link here*
+
+Step 1: Installing the Flocker client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+****
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+************
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+To test your installation, run the following to check that you have the Flocker client installed correctly:
+
+.. prompt:: bash [you@laptop:~$]
+
+   flocker-deploy --version
+
+Successful installation will display the version of Flocker.
+
+Step 2: Installing Flocker on your hosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Option A: Installing Flocker on local VMs
+*****************************************
+
+.. note:: You must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_  installed to create the VMs and start the containers for this tutorial.
+
+In Step 1 you installed the Flocker client on your local machine.
+For the next step in this tutorial you now need two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies) on the hosts, run the following command and Vagrant will create the environments you need:
+
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key && \
+   vagrant up && \
+   [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+   ssh-add ~/.vagrant.d/insecure_private_key
+
+Option B: Installing Flocker on AWS or Rackspace
+************************************************
+
+The two instances of Flocker each run on a separate host.
```

Comment:
This document has hosts and nodes and servers. Pick one, probably node as we have `flocker-node`.